# RequestDock.srv
---
bool success
string message